src/pages/wallet/WalletStatementPage.js

```diff
@@ -71,7 +78,7 @@ class WalletStatementPage extends React.Component {
             return;
         }
 
-        Growl.show(this.props.translate('statementPage.generatingPDF'), CONST.GROWL.NOTICE, 5000);
+        Growl.show(this.props.translate('statementPage.generatingPDF'), CONST.GROWL.SUCCESS, 3000);
```

Comment:
I suppose we'll get rid of the Growl alerts eventually (maybe by using an optimistic message?). But since it's fixing a crash I don't think it's a blocker